app/scripts/modules/core/projects/dashboard/pipeline/ProjectPipeline.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
```

Comment:
We've been avoiding `*` imports. Is it more conventional in React to import it this way, rather than `import {Component} from 'react'`?